spec-main/api-browser-window-spec.ts

```diff
@@ -3660,11 +3660,12 @@ describe('BrowserWindow module', () => {
         expect(w.webContents.isLoadingMainFrame()).to.be.false('isLoadingMainFrame');
 
         const didStartLoading = emittedOnce(w.webContents, 'did-start-loading');
-        w.webContents.executeJavaScript(`
+        await w.webContents.executeJavaScript(`(() => {
           var iframe = document.createElement('iframe')
           iframe.src = '${serverUrl}/page2'
           document.body.appendChild(iframe)
-        `);
+          return null
+        })()`);
```

Comment:
I don't think the IIFE is needed here, just write `null` at the end of the block.

```suggestion
        await w.webContents.executeJavaScript(`
          var iframe = document.createElement('iframe')
          iframe.src = '${serverUrl}/page2'
          document.body.appendChild(iframe)
          null
        `);
```


... also, should this be split into a separate PR? it seems unrelated.